frontend/src/main/components/OurTable.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useTable, useSortBy } from 'react-table'
+import { Table } from "react-bootstrap";
+
+export default function OurTable({ columns, data }) {
```

Comment:
Is the convention going forward to prefix our components with `Our`? We don't have any guidance on component naming conventions right now so I just wanted to clarify.